src/coreclr/pal/src/init/pal.cpp

```diff
@@ -673,13 +673,6 @@ Initialize(
             goto CLEANUP15;
         }
 
-        if (FALSE == NUMASupportInitialize())
```

Comment:
assume its fine to remove the Numa initialization from the pal, since its now being handled with the gc_unix pal.